extensions/podman/src/podman-cli.ts

```diff
@@ -89,7 +89,7 @@ export function execPromise(
     token?.onCancellationRequested(() => {
       process.kill();
       // reject the promise
-      reject('Execution cancelled');
+      resolve('Execution cancelled');
```

Comment:
is it a reject or resolve ?

Reply:
not 100% but i thought that if the user is cancelling it voluntarily it's not an error, you want to resolve it without doing anything. If i reject the user will see the error message and i would need to filter it in the UI but still ... this is not an error.

Reply:
I suppose I expect an error because the method is shared by a lot of code and client needs to know what happened as it did not finished successfully
but error should be like a CancellationRequestError so it's filtered out by client

Reply:
Else, what we could do is always return a complex object with helpers like `isCancelled`

so instead of returning a string, we return an object that says if it was cancelled, succeed, error, etc.

 it could be an improvement done with https://github.com/containers/podman-desktop/issues/1585